README.md

```diff
@@ -10,20 +10,18 @@ using the [STOMP protocol](http://stomp.github.io/) or via the ARGO Messaging Se
 Messages are signed and may be encrypted during transit.
 Persistent queues should be used to guarantee delivery.
 
-SSM is written in Python. Packages are available for RHEL 6 and 7, and
- Ubuntu Trusty.
+SSM is written in Python. Packages are available for RHEL 7, and Ubuntu Trusty.
 
 For more information about SSM, see the [EGI wiki](https://wiki.egi.eu/wiki/APEL/SSM).
 
 ## Acknowledgements
 
 <span>
-  <img alt="STFC logo" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/images/UKRI_STF_Council-Logo_Horiz-RGB_crop.png" height="57" />
-  <img alt="EU flag" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/images/eu_flag_yellow_low_150.png" height="51" />
-  <img alt="EOSC-hub logo" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/images/eosc-hub-v-web_150.png" height="57" />
+  <img alt="STFC logo" src="https://github.com/GOCDB/gocdb/blob/dev/htdocs/images/logos/ukri_stfc.png" height="57" />
```

Comment:
Thinking about this, I realised we could provide permanent links by using the commit reference in the URL. Could you update the links to these, please?

https://github.com/GOCDB/gocdb/raw/a3df819/htdocs/images/logos/ukri_stfc.png
https://github.com/GOCDB/gocdb/raw/a3df819/htdocs/images/flags/eu.png